utils/browser/src/lib.rs

```diff
@@ -32,20 +32,40 @@ use sc_chain_spec::Extension;
 use libp2p_wasm_ext::{ExtTransport, ffi};
 
 pub use console_error_panic_hook::set_once as set_console_error_panic_hook;
-pub use console_log::init_with_level as init_console_log;
+
+/// Initialize the logger and return a `Telemetries` and a wasm `ExtTransport`.
+///
+/// The `Telemetries` object can be use to create `Telemetry` objects and the wasm `ExtTransport`
```

Comment:
You need to pass the ExtTransport to the config to create a node. Some network things depend on this. But the ExtTransport is also used by the Telemetry (because of the network connection to the telemetry server).

I was thinking that maybe having a global static variable for that would be easier though.... at the price of, well, global variables XD

cc @tomaka

Reply:
Please don't use global variables.
Maybe that calling `ExtTransport::new(ffi::websocket_transport())` looks like a heavy operation, but I think that the `ExtTransport` is literally 0 bytes. In other words, it's the same as `()`.
Rather than creating it here and passing it around, you can just recreate it as many as time as you want everywhere you need it.

Reply:
What I mean by my comment is that you write "can be used". That sounds like it is not required at all and we don't need to return it here.

Reply:
> Rather than creating it here and passing it around, you can just recreate it as many times as you want anywhere you need it.

That would be cool. So you mean I can delete it from Configuration and entirely from telemetry and instead I re-create it in the few places needed if `cfg(wasm..something)`?

> What I mean by my comment is that you write "can be used". That sounds like it is not required at all and we don't need to return it here.

Ah yeah right you mean the doc is wrong XD Yes it's required you're right

Reply:
Ok so I made 2 commits to remove passing around the ExtTransport:

1. for sc-telemetry: 32b0b14ddd81974b5286c80c1fe5c787474f927e
2. for sc-network: 834eac41a175aff8f4831b98817f2da030ca6de0

@tomaka let me know if this is ok or if I should revert the sc-network

Reply:
Following tomaka's advice: 5140912da